pycobertura/reporters.py

```diff
@@ -202,6 +214,7 @@ def get_report_lines(self):
 
 class TextReporterDelta(DeltaReporter):
     def __init__(self, *args, **kwargs):
+        self.not_available = "-"
         super(TextReporterDelta, self).__init__(*args, **kwargs)
```

Comment:
Can we have this on the class level since it's not supposed to change on a per-instance basis?

```suggestion
class TextReporterDelta(DeltaReporter):
    not_available = "-"
```

Also, can we remove the `__init__()` method since we're not doing anything useful in it anymore?

Reply:
done